libraries/go/webhook.go

```diff
@@ -114,7 +115,8 @@ func (wh *Webhook) verify(payload []byte, headers http.Header, enforceTolerance
 			return nil
 		}
 	}
-	return errNoMatchingSignature
+
+	return fmt.Errorf("unable to verify payload, err: %w", ErrNoMatchingSignature)
```

Comment:
why don't we return `ErrRequiredHeaders` ? It would allow the consumer to do error comparison